### ISSUE TITLE
Do not allow worker names with bad characters in them

### ISSUE THREAD

**Issue by PeterDSteinberg** [Anaconda-Server/anaconda-build#186]
When doing:
```
anaconda worker register username/queue --name some_worker

```
, there can be downstream problems if `some_worker` has bad characters such as a `/`.

It would probably be good to similarly review other arguments to build_commands and worker_commands.

**Comment by srossross** [comment]
@PeterDSteinberg I think this issue should be in "In Progress" and the corresponding PR should be in "Ready for Review"

**Comment by dsludwig** [comment]
Fixed by #214
